app/models/issue.rb

```diff
@@ -15,9 +15,24 @@ def valid_for_user?(user, skip_update = Rails.env.test?)
     true
   end
 
+  def fetcher
+    @fetcher ||= GithubFetcher::Issue.new(
+      owner_name: owner_name,
+      repo_name: repo_name,
+      number: number,
+    )
+  end
+
+  def comments_fetcher
+    @fetcher ||= GithubFetcher::IssueComments.new(
```

Comment:
No problem. And apologies for the random comment—I happened to come across this PR in my twitter feed when @nateberkopec. Cool refactor!